app/services/create_account.rb

```diff
@@ -41,6 +42,10 @@ def create_redis_namespace
     CreateRedisNamespaceJob.perform_later(account)
   end
 
+  def create_default_admin_set
+    CreateDefaultAdminSetJob.perform_later(account)
```

Comment:
After talking this over with @cbeer I can't think of a good way to do this.  We need our jobs to be idempotent (i.e. small) so they can fail and be retried, but we also need to have dependency management of jobs. I don't know what the solution is, but I don't think this is the place to add it.